examples/meta/src/evaluation/cross_validation_pipeline.sg

```diff
@@ -0,0 +1,33 @@
+File f_feats_train = csv_file("../../data/classifier_binary_2d_linear_features_train.dat")
+File f_feats_test = csv_file("../../data/classifier_binary_2d_linear_features_test.dat")
+File f_labels_train = csv_file("../../data/classifier_binary_2d_linear_labels_train.dat")
+File f_labels_test = csv_file("../../data/classifier_binary_2d_linear_labels_test.dat")
+
+#![create_features]
+Features feats_train = features(f_feats_train)
+Features feats_test = features(f_feats_test)
+Labels labels_train = labels(f_labels_train)
+Labels labels_test = labels(f_labels_test)
+#![create_features]
+
+#![create_pipeline
+Transformer subMean = transformer("PruneVarSubMean")
+Machine svm = machine("LibLinear")
+
+PipelineBuilder builder()
```

Comment:
Same here, could you pls use a factory for this. We dont want to use constructors in the examples

Reply:
could you explain how to use factory here?
we could create a factory `PipelineBuilder* pipeline_builder()`, or?

Reply:
yes exactly, though I would just name it `pipeline`